### PR TITLE
Update README.md for WSL2 instructions in Windows installation section

### PR DESCRIPTION
**Pull request by Unrooted**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,11 +211,25 @@ $ sudo apt-get autoremove -y
 
 Configure the `DISPLAY` environment variable for the X server, and load the changes in bash:
 
+* For WSL1:
```
Comment (cdepillabout) [inline README.md:214]:
```suggestion
For WSL1:
```

File: README.md [hunk 2]
```diff
@@ -211,11 +211,25 @@ $ sudo apt-get autoremove -y
...
+
 ```console
 $ echo "export DISPLAY=localhost:0.0" >> ~/.bashrc
 $ source ~/.bashrc
 ```
 
+* For WSL2:
```
Comment (cdepillabout) [inline README.md:221]:
```suggestion
For WSL2:
```

File: README.md [hunk 3]
```diff
@@ -211,11 +211,25 @@ $ sudo apt-get autoremove -y
...
+
+```console
+$ echo export DISPLAY=$(awk '/nameserver / {print $2; exit}' /etc/resolv.conf 2>/dev/null):0 >> ~/.bashrc
+$ echo export LIBGL_ALWAYS_INDIRECT=1 >> ~/.bashrc
+$ source ~/.bashrc
+```
+
+> If you're using WSL2, you have to create a separate **inbound rule** for TCP port 6000, to allow WSL access to the X server.
```
Comment (cdepillabout) [inline README.md:229]:
```suggestion
If you're using WSL2, you have to create a separate **inbound rule** for TCP port 6000, to allow WSL access to the X server.
```

**Comment by cdepillabout** [comment]
Thanks!

**Comment by Unrooted** [comment]
No problem.

**Pull request merged**